metadata-ingestion/tests/integration/druid/test_druid.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from freezegun import freeze_time
+
+from tests.test_helpers import mce_helpers
+from tests.test_helpers.click_helpers import run_datahub_cmd
+from tests.test_helpers.docker_helpers import wait_for_port
+
+FROZEN_TIME = "2025-02-24 09:00:00"
+
+
+@pytest.fixture(scope="module")
+def test_resources_dir(pytestconfig):
+    return pytestconfig.rootpath / "tests/integration/druid"
```

Comment:
Lately I've been preferring to use `_resource_dir = pathlib.Path(__file__).parent`

Reply:
commit [c27191d](https://github.com/datahub-project/datahub/pull/12717/commits/c27191d8a233cd7ec7173918346caa03a17f80c9)